src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -287,6 +294,11 @@ public async Task<int> DeployAsync()
                                 await CreateDefaultStorageContainersAsync(storageAccount);
                             }
 
+                            if ((installedVersion is null || installedVersion < new Version(4, 5)) && string.IsNullOrWhiteSpace(settings["BatchNodesSubnetId"]))
```

Comment:
Use (4, 4). Otherwise when upgrading from 4.4 to 4.5 it will repeat this update.

Reply:
Fixed